elementalconductor/presets.go

```diff
@@ -0,0 +1,70 @@
+package elementalconductor
+
+// GetPresets returns a list of presets
+func (c *Client) GetPresets() (*PresetList, error) {
+	var result *PresetList
+	err := c.do("GET", "/presets", nil, &result)
+	if err != nil {
+		return nil, err
+	}
+	return result, nil
+}
+
+// GetPreset return details of a given presetID
+func (c *Client) GetPreset(presetID string) (*Preset, error) {
+	var result *Preset
+	err := c.do("GET", "/presets/"+presetID, nil, &result)
+	if err != nil {
+		return nil, err
+	}
+
+	return result, nil
+}
+
+// CreatePreset creates a new preset
+func (c *Client) CreatePreset(p *Preset) (*Preset, error) {
```

Comment:
I noticed that we used ``PostJob`` for jobs and ``CreatePreset`` for presets, and ``CreateX`` looks better. I know it has nothing to do with your PR, but can you rename the PostJob method to CreateJob? Feel free to send it in another PR :-)